### DISCUSSION TITLE
[ios]1.25.0 podspec file need update

### DISCUSSION THREAD

**Discussion by liangjie90** [google/filament#5970] [Q&A]
The Podspec file was not updated and the reference could not be found

**Comment by bejado** [comment]
1.25.0 was released last month on [CocoaPods](https://github.com/CocoaPods/Specs/commit/a64b56e9d6a0e9c00f1c840bac206a38ab42219a).

**Reply by liangjie90** [reply]
pod Filament', '1.25.0 can not find <gltfio/materials/uberarchive.h> but in the 
 filament-v1.25.0-ios.tgz  has this file
